pkg/tsdb/azuremonitor/utils/utils.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend/log"
+	"github.com/grafana/grafana/pkg/tsdb/azuremonitor/types"
+)
+
+func GetFirstSubscriptionOrDefault(ctx context.Context, dsInfo types.DatasourceInfo, logger log.Logger) (string, error) {
+	if dsInfo.Settings.SubscriptionId != "" {
+		return dsInfo.Settings.SubscriptionId, nil
+	}
+
+	subscriptionsApiVersion := "2020-01-01"
```

Comment:
nit this should be a `const`

Reply:
Have converted this to a `const` and reused in one other place